docs/source/plotting/MeshPlotHelp.rst

```diff
@@ -76,7 +81,294 @@ Basic example of plotting a general Poly3DCollection `Polygon <https://matplotli
+      z_range = abs(z_limits[1] - z_limits[0])
+      z_middle = np.mean(z_limits)
+
+      plot_radius = 0.5*max([x_range, y_range, z_range])
+
+      ax.set_xlim3d([x_middle - plot_radius, x_middle + plot_radius])
+      ax.set_ylim3d([y_middle - plot_radius, y_middle + plot_radius])
+      ax.set_zlim3d([z_middle - plot_radius, z_middle + plot_radius])
+
+   cuboid = " \
+   <cuboid id='some-cuboid'> \
+   <height val='2.0'  /> \
+   <width val='2.0' />  \
+   <depth  val='0.2' />  \
+   <centre x='10.0' y='10.0' z='10.0'  />  \
+   </cuboid>  \
+   <algebra val='some-cuboid' /> \
+   "
+
+   ws = CreateSampleWorkspace()
+   SetGoniometer(ws, Axis0="120,0,0,1,1")
+   SetSample(ws, Geometry={'Shape': 'CSG', 'Value': cuboid})
+   sample = ws.sample()
+
+   SetUB(ws, a=1, b=1, c=2, alpha=90, beta=90, gamma=60)
+   if not sample.hasOrientedLattice():
+      raise Exception("There is no valid lattice")
+
+   UB = np.array(ws.sample().getOrientedLattice().getUB())
+   hkl = np.array([[1.0,0.0,0.0],[0.0,1.0,0.0],[0.0,0.0,1.0]])
+   QSample = np.matmul(UB,hkl)
+   Goniometer = ws.getRun().getGoniometer().getR()
+   reciprocal_lattice = np.matmul(Goniometer,QSample)#QLab
+   real_lattice = (2.0*np.pi)*np.linalg.inv(np.transpose(reciprocal_lattice))
+
+   shape = sample.getShape()
+   mesh = shape.getMesh()
+
+   facecolors = ['purple','mediumorchid','royalblue','b','red','firebrick','green', 'darkgreen','grey','black', 'gold', 'orange']
+   mesh_polygon = Poly3DCollection(mesh, facecolors = facecolors, linewidths=0.1)
+
+   fig, axes = plt.subplots(subplot_kw={'projection':'mantid3d'})
+   axes.add_collection3d(mesh_polygon)
+
+   axes.set_title('Sample Shape: Cuboid {} \n with Real and Reciprocal lattice vectors'.format(ws))
+   axes.set_xlabel('X / m')
+   axes.set_ylabel('Y / m')
+   axes.set_zlabel('Z / m')
+
+   axes_lims = mesh.flatten()
+   axes.auto_scale_xyz(axes_lims[0::3], axes_lims[1::3], axes_lims[2::3])
+   set_axes_equal(axes)
+   axes.view_init(elev=12, azim=44)
+
+   def arrow(ax, vector, origin = None, factor = None, color = 'black',linestyle = '-'):
+      if origin == None:
+         origin = (ax.get_xlim3d()[1],ax.get_ylim3d()[1],ax.get_zlim3d()[1])
+      if factor == None:
+         lims = ax.get_xlim3d()
+         factor = (lims[1]-lims[0]) / 3.0
+      vector_norm = vector / np.linalg.norm(vector)
+      ax.quiver(
+            origin[0], origin[1], origin[2],
+            vector_norm[0]*factor, vector_norm[1]*factor, vector_norm[2]*factor,
+            color = color,
+            linestyle = linestyle
+      )
+
+   colors = ['r','g','b']
+   for i in range(3): # plot real_lattice with '-' solid linestyle
+      arrow(axes, real_lattice[:,i], color = colors[i])
+   for i in range(3): # plot reciprocal_lattice with '--' dashed linestyle
+      arrow(axes, reciprocal_lattice[:,i], color = colors[i], linestyle = '--')
+
+   plt.show()
+
+For more plotting advice: :ref:`02_scripting_plots`
 
 |
 |
```

Comment:
The figure shown under the Options menu heading shows two tabs: Axes and Images. I only see the Axes tab

Reply:
Great spot, I am removing this as it more general to all 3D plots